ctwrap/simulation.py

```diff
@@ -168,6 +198,25 @@ class SimulationHandler(object):
     """
     Class handling parameter variations.
     Class adds methods to switch between multiple configurations.
+
+    .. note:: :class:`SimulationHandler` objects should be
+        instantiated using the :meth:`from_yaml`.
+
+    An example of a minimal yaml input is:
+
+       .. code-block:: YAML
+
+          output: # output file information
+            format: null # no file output
```

Comment:
We should probably drop this as it is superseded by the `save` methods that are now moved to the simulation modules.

Reply:
I understand while we need to drop `format` but  `output` should not be entirely dropped so the user can specify `filename`

Reply:
ok.